### PR TITLE
Increase image size

### PR DESCRIPTION
**Pull request by shaardie**
seems like our image is too small to be easy to understand. This patch
increases the image size.

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -46,7 +46,7 @@
 }
 
 #title img {
-  width: 70px;
+  width: 140px;
```
Comment (lkiesow) [inline assets/css/style.scss]:
```suggestion
  width: 120px;
```

File: assets/css/style.scss [hunk 2]
```diff
@@ -46,7 +46,7 @@
...
   margin-bottom: 20px;
```
Comment (lkiesow) [inline assets/css/style.scss]:
```suggestion
  margin-bottom: 40px;
```

**Review by lkiesow — COMMENTED** [review]
What do you think about these slight modifications?

**Comment by shaardie** [comment]
I am okay with your suggestions. :+1:

**Pull request merged**